test/activities/content/ContentEntity.js

```diff
@@ -0,0 +1,23 @@
+/* global fetchMock */
+
+import { ContentEntity } from '../../../src/activities/content/ContentEntity.js';
+import { editableContent } from './data/EditableContent.js';
+
+describe('ContentEntity', () => {
+	let editableEntity;
+
+	beforeEach(() => {
+		editableEntity = window.D2L.Hypermedia.Siren.Parse(editableContent);
+	});
+
+	afterEach(() => {
+		fetchMock.reset();
```

Comment:
what does `fetchMock` do in this file? do we need to reset it after every test?

Reply:
It's not actually doing anything for us right now. It was a copy-paste from the ActivityEntity test file. Will remove it for now while we don't need it.